test/powershell/Language/Operators/ComparisonOperator.Tests.ps1

```diff
@@ -82,6 +82,22 @@ Describe "ComparisonOperator" -Tag "CI" {
         param($lhs, $operator, $rhs)
         Invoke-Expression "$lhs $operator $rhs" | Should -BeFalse
     }
+
+    It "Should be <result> for <lhs> <operator> <rhs>" -TestCases @(
+        @{ lhs = 'abc`def'; operator = '-like'; rhs = 'abc`def'; result = $true }
+        @{ lhs = 'abc``def'; operator = '-like'; rhs = 'abc``def'; result = $true }
+        @{ lhs = '`abcdef'; operator = '-like'; rhs = '`abcdef'; result = $true }
+        @{ lhs = 'abcdef`'; operator = '-like'; rhs = 'abcdef`'; result = $true }
```

Comment:
Testing `-notlike` doesn't add value but what about `-match`? Do we fix it too?
Also maybe add something like 'abcdef`n' to check we don't convert '`n' to CR.

Reply:
I don't think -match is relevant since -like maps to WildcardPattern.IsMatch. I would assume -match maps to Regex.IsMatch.

Reply:
Tests must protect from regressions. If we change `-match` implementation we can get a regression. We can't just assume that both of these operators are based on the same method.

Reply:
@iSazonov Fair enough. I have added some backtick tests for -match.